packages/@aws-cdk/aws-backup/README.md

```diff
@@ -218,3 +236,27 @@ const role = new iam.Role(this, 'Access Role', { assumedBy: new iam.ServicePrinc
 
 importedVault.grant(role, 'backup:StartBackupJob');
 ```
+
+## Backup selection IAM role policies
+
+When creating a new `BackupSelection`, if no user IAM role is provided, a new
```

Comment:
Why `user` in the sentence? Can it not be skipped completely?

Reply:
Good catch